ui/src/features/auth/oidc-login.tsx

```diff
@@ -106,42 +119,57 @@ export const OIDCLogin = ({ oidcConfig }: Props) => {
         searchParams.delete('state');
       }
 
-      const params = oauth.validateAuthResponse(as, client, searchParams, oauth.expectNoState);
+      try {
+        const params = validateAuthResponse(as, client, searchParams);
+
+        const response = await authorizationCodeGrantRequest(
+          as,
+          client,
+          oidcClientAuth,
+          params,
+          redirectURI,
+          code_verifier,
+          {
+            [allowInsecureRequests]: shouldAllowHttpRequest(),
+            additionalParameters: [['client_id', client.client_id]]
```

Comment:
will merge after checking whether we need more than `client_id` in parameters in advanced scenarios